tests/playTests.sh

```diff
@@ -409,6 +409,34 @@ println "compress multiple files including a missing one (notHere) : "
 $ZSTD -f tmp1 notHere tmp2 && die "missing file not detected!"
 
 
+println "\n===>  size-hint mode"
+
```

Comment:
We should test with a dictionary `-D` (search for other uses), and when compressing multiple files.

We should test that compressing with `zstd --size-hint 1k tmp1 tmp2` is the same as compressing each file individually with the same size hint.